docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,59 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |find a person by tags |locate people easily
+
+|`* * *` |user |sort people by name |locate people easily
+
+|`* * *` |user |set a list of favourite people|
+
+|`* * *` |user |see a list of frequently accessed contacts|
```

Comment:
Again it would give more clarity on why there should be a list of frequently accessed contacts. Perhaps so that you can have a faster way to contact these contacts?
Recall from tutorial on why we should specify user some more, in this case perhaps a busy user would find this feature useful?